app/models/refinery/search_engine.rb

```diff
@@ -2,18 +2,23 @@ module Refinery
   class SearchEngine
 
     # How many results should we show per page
-    RESULTS_LIMIT = 10
+    RESULTS_LIMIT = 10 unless defined? RESULTS_LIMIT
```

Comment:
Could you create a configuration instead of RESULTS_LIMIT constant ?

Like in refinery-blog : https://github.com/refinery/refinerycms-blog/blob/master/lib/refinery/blog/configuration.rb